includes/admin/settings/class-alnp-settings-events.php

```diff
@@ -87,9 +85,7 @@ public function get_settings() {
 		 * @since  1.5.0
 		 */
 		public function output() {
-			$settings = $this->get_settings();
```

Comment:
Removing this to add it directly does not always work in some PHP versions. Please leave as is so it works for all PHP versions. Thank you.